src/main/cluster.ts

```diff
@@ -245,7 +245,7 @@ export class Cluster implements ClusterModel, ClusterState {
    * Kubernetes version
    */
   get version(): string {
-    return String(this.metadata?.version) || "";
+    return String(this.metadata?.version || "");
```

Comment:
Before this change this getter resulted in `undefined` being output (why?) when the cluster in question was not active

Reply:
because `String(undefined) === "undefined"`

Reply:
ah, of course. JavaScript. I guess that could've been the expected behaviour? Probably not.

Reply:
```js
console.log(String())
console.log(String(undefined))
console.log(String(""))
console.log(String(0))
console.log(String(false))
console.log(String(null))
```

produces:
```js
""
"undefined"
""
"0"
"false"
"null"
```

Reply:
Why we ever need to wrap cluster version into `String()`? Can it have any other type?

Reply:
I guess just by virtue of `this.metadata`'s generic definition?

```
export interface ClusterMetadata {
  [key: string]: string | number | boolean | object;
}
```

Reply:
```suggestion
    return String(this.metadata?.version ?? "");
```

So that if for whatever reason `version` is `false` or `0` that is passed through.